DANS-KNAW/easy-deposit
# DD-408: easy-sword2: alternative authentication mode through user.properties

Fixes DD-408

#### When applied 
* there is another option for `authorization`: through `users.properties` file


#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?
- in deasy set `auth.mode=file` in `application.properties` (/etc/opt/dans.knaw.nl/easy-sword2/application.properties)
- call easy-sword2 with `user001` and pw `user001`, e.g. `./run.sh Simple https://deasy.dans.knaw.nl/sword2/collection/1 user001 user001 src/main/resources/agreement-flow/valid/audiences/`
- check the log (/var/opt/dans.knaw.nl/log/easy-sword2/easy-sword2.log)

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
